sawtooth/cli.py

```diff
@@ -309,6 +356,52 @@ def do_submit(args):
         if not client.wait_for_commit():
             raise CliException("transaction was not successfully committed")
 
+class SawtoothCtl(_Communication):
```

Comment:
Would prefer we use SawtoothClient directly, similar to do_submit().  Our intent with the underscore in _Communication is that it be private; at some point we will remove _Communication as we further refine the SawtoothClient class over time, so we are considering it an implementation detail.